pkg/azureupdate/admit_azureclusterconfig.go

```diff
@@ -0,0 +1,109 @@
+package azureupdate
```

Comment:
shouldn't this file called validate_azureclusterconfig.go?

Reply:
same as the other files?

Reply:
makes sense as well, done